mise.toml

```diff
@@ -3,7 +3,7 @@
     swiftlint = "0.59.1"
     swiftformat = "0.57.2"
     pnpm = "10.14"
-    node = "20.12.0"
```

Comment:
Don't forget to rebase from `main` since this file lives now in `server/mise.toml`